modules/nw-cluster-network-range-edit.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-cluster-network-range.adoc
+
+:_content-type: PROCEDURE
+[id="nw-cluster-network-range-edit_{context}"]
+= Expanding the cluster network IP address range
+
+You can expand the IP address range for the cluster network. Because this change requires rolling out a new Operator configuration across the cluster, it can take up to 30 minutes to take effect.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+* Log in to the cluster with a user with `cluster-admin` privileges.
+* Ensure that the cluster uses the OVN-Kubernetes network plugin.
```

Comment:
s/ plugin / plug-in according to IBMSG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#plug-in